src/bin/wasmtime.rs

```diff
@@ -47,17 +46,19 @@ enum Subcommand {
     #[cfg(feature = "cache")]
     Config(wasmtime_cli::commands::ConfigCommand),
     /// Compiles a WebAssembly module.
-    Compile(CompileCommand),
+    #[cfg(feature = "cranelift")]
+    Compile(wasmtime_cli::commands::CompileCommand),
```

Comment:
Would be nice to add some empty lines between these variant definitions now that they are getting a bit noisier with all the `cfg`s and all that.